test/ollama-gpu-cleanup.test.js

```
@@ -0,0 +1,251 @@
+                      path: "/api/generate",
+                      method: "POST",
+                      timeout: 3000,
+                      headers: { "Content-Type": "application/json" },
+                    },
+                    () => {},
+                  );
+                  unloadReq.on("error", () => {});
+                  unloadReq.write(JSON.stringify({ model: entry.name, keep_alive: 0 }));
+                  unloadReq.end();
+                }
+              } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)